packages/client/src/components/atoms/SlotOperationButtons/CopyButton.tsx

```diff
@@ -98,15 +102,20 @@ export const CopyButton: React.FC = () => {
     }),
   };
 
+  const testIdLookup = {
+    [ButtonContextType.Day]: `${__copyDayButtonId__}${date}`,
+    [ButtonContextType.Week]: __copyWeekButtonId__,
+  };
```

Comment:
Not sure if this amount of specificity is necessary (I did it for aria-labels as it will provide some more context to a screen reader)...I'll get back to this

Reply:
Right...it's necessary to differentiate between multiple buttons on the page in e2e 👍